### ISSUE TITLE
`ee stack restart` doesn't consider remote mysql

### ISSUE THREAD

**Issue by rahul286** [rtCamp/easyengine#428]
```
> ee stack restart 
Restart : nginx     [OK]
Restart : php5-fpm  [OK]
Restart : mysql     [Failed]
Restart : postfix   [OK]
```

**Comment by MiteshShah** [comment]
@gau1991 As we discuss earlier we have to check whether mysql-server or percona-server-server package is installed or not.

Also you can check /var/lib/mysql directory to cross check mysql is local or remote.

**Comment by rahul286** [comment]
I think `/var/lib/mysql` is not foolproof. MySQL config must have way to override storage location.

**Comment by gau1991** [comment]
Fixed with EasyEngine 3.0.3
